### ISSUE TITLE
Admin Interface Main Menu Focus Indicator Inadequate

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1776]
### Steps to Reproduce

1. Navigate to the admin interface for a book
2. Use the keyboard to navigate to the Main Menu navigation region
3. Use the keyboard to navigate through the options
4. Notice that the focus indicator has the exact same styling as indicator for the current page

**Expected behavior:** 
The focus indicators should be different.
**Actual behavior:** 
The focus indicators are the same. This could potentially be a big issue for a keyboard user with low visibility or someone with a cognitive disability.

**Comment by avaaec** [comment]
This issue is in reference to the Pressbooks A11y admin interface theme. The default Pressbooks theme also has inadequate focus indication for this menu.

**Comment by laurenray** [comment]
There are two different color schemes and the one for Pressbooks A11y admin interface theme is less accessible. This is a UW priority 1.

**Comment by SteelWagstaff** [comment]
If I'm understanding this correctly, the suggestion is to visually disambiguate the 'current' page and the focus indicator/hover state for the admin menu in the a11y admin interface. For example, in the screenshot below a user is on the Book Info page (indicated by a bold red link) and has keyboard navigated so that Export->Export is in focus/hover state. That link appears visually similar (bold, red) to the current page and might be confusing to a user. Here's what I'm seeing:
![Screenshot from 2019-10-18 17-25-12](https://user-images.githubusercontent.com/13485451/67131927-4d0f9500-f1cc-11e9-8f33-a71ffcb7fd10.png)

**Comment by SteelWagstaff** [comment]
Values set here: https://github.com/pressbooks/pressbooks/blob/master/assets/src/styles/colors-pb-a11y.scss. To disambiguate, recommended values for #adminmenu li.current are `background-color: #111` and `color: #fff`

**Comment by connermwallace** [comment]
University of Washington Accessibility Technology Services has inspected this issue and we have verified that it has been fixed. Version: 5.3.2.
